.gitmodules

```diff
@@ -31,3 +31,12 @@
 [submodule "external/alfons"]
 	path = external/alfons
 	url = https://github.com/hjanetzek/alfons
+[submodule "external/curl"]
+	path = external/curl
+	url = https://github.com/lygstate/curl.git
+[submodule "external/zlib"]
+	path = external/zlib
+	url = https://github.com/madler/zlib.git
+[submodule "external/angle"]
+	path = external/angle
+	url = https://github.com/lygstate/angle.git
```

Comment:
Does windows not have some networking library support?

We have networking implemented for every platform independently using the libraries available on that specific platform. This gives us the flexibility of using platform library specific features like, caching etc. We have `curl` for linux and raspberry-pi since thats the best library we can use there. For `OSX` and `iOS` we use `NSurl` and for `Android` we use `OkHTTP`.